controllers/index.js

```diff
@@ -49,3 +50,12 @@ exports.getUsers = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getComments = (req, res, next) => {
```

Comment:
getCommentsById? this is a more accurate description of the behaviour of this function